CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-### 2.2.1 (Next)
+### 2.3.0 (Next)
 
+* [#87](https://github.com/dblock/strava-ruby-client/pull/87): Prepares v2.3.0 by improving the event specs - [@simonneutert](https://github.com/simonneutert).
```

Comment:
I'd remove this, it doesn't tell much :)

Reply:
something is required, else DangerBot will be unhappy 🤷‍♂️

Reply:
It's not fatal though.